submissions/AnnaStoyano/oop-exercise/index.js

```diff
@@ -0,0 +1,112 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Mammals{
+   constructor(name,gender,friends){
+      this.species = 'mammals',
+      this.name = name,
+      this.gender = gender,
+      this.legs = 4,
+      this.friends = friends
+   }
+}
+
+class Dog extends Mammals{
+   constructor(name,gender,friends,saying){
+      super(name,gender,friends);
+      this.species = 'dog',
+      this.saying = saying || 'guf'
+   }
+};
+
+class Cat extends Mammals{
+   constructor(name,gender,friends,saying){
+      super(name,gender,saying,friends);
+      this.species = 'cat',
+      this.saying = saying || 'guf'
+   }
+
+   getSaying(){
+      return this.saying || 'meow';
+   }
+}
+
+class Human extends Mammals{
+   constructor(name,gender,friends,saying){
+      super(name,gender,friends);
+      this.species = 'human'
```

Comment:
Having no semicolon is generally valid in JS. Yet we recommend to keep code style consitent.